mms_message.go

```diff
@@ -0,0 +1,47 @@
+package messagebird
+
+import (
+    "net/url"
+    "time"
+)
+
+type MmsMessage struct {
+    Id                string
+    HRef              string
+    Direction         string
+    Originator        string
+    Body              string
+    Reference         string
+    Subject           string
+    MediaUrls         []string
+    ScheduledDatetime *time.Time
+    CreatedDatetime   *time.Time
+    Recipients        Recipients
+    Errors            []Error
+}
+
+type MmsMessageParams struct {
+    Subject           string
+    Reference         string
+    ScheduledDatetime time.Time
+}
+
+func paramsForMmsMessage(params *MmsMessageParams) (*url.Values, error) {
```

Comment:
This function does not need to return an `error`, because nothing in this function generates an error. That is to say, unless you move the `if body == "" && mediaUrls == nil` from `NewMmsMessage()` here.